app/assets/javascripts/discourse/controllers/forgot-password.js.es6

```diff
@@ -29,45 +29,37 @@ export default Ember.Controller.extend(ModalFunctionality, {
     },
 
     resetPassword() {
-      return this._submit('/session/forgot_password', 'forgot_password.complete');
-    },
+      if (this.get('submitDisabled')) return false;
+      this.set('disabled', true);
 
-    emailLogin() {
-      return this._submit('/u/email-login', 'email_login.complete');
-    }
-  },
-
-  _submit(route, translationKey) {
-    if (this.get('submitDisabled')) return false;
-    this.set('disabled', true);
-
-    ajax(route, {
-      data: { login: this.get('accountEmailOrUsername').trim() },
-      type: 'POST'
-    }).then(data => {
-      const escaped = escapeExpression(this.get('accountEmailOrUsername'));
-      const isEmail = this.get('accountEmailOrUsername').match(/@/);
-      let key = `${translationKey}_${isEmail ? 'email' : 'username'}`;
-      let extraClass;
+      // success is not using flash, so make sure we are clearing it
```

Comment:
Perhaps this comment could be replaced with a test case? :thought_balloon:

Reply:
this is way out of the commit scope already and also I feel like the good fix would be to just get rid of this "offerHelp" part.